apps/emqx/src/emqx_cm.erl

```diff
@@ -448,20 +448,23 @@ kick_session(Action, ClientId, ChanPid) ->
                           , action => Action
                           , error => Error
                           , reason => Reason
-                          })
+                          },
+                #{clientid => unicode:characters_to_list(ClientId, utf8)})
```

Comment:
Is it possible not to convert client id before the allow check?

Reply:
It is already  after logger:allow/2
```erlang
%% structured logging, meta is for handler's filter.
-define(SLOG(Level, Data, Meta),
%% check 'allow' here, only evaluate Data when necessary
    case logger:allow(Level, ?MODULE) of
        true ->
            logger:log(Level, (Data), (Meta#{ mfa => {?MODULE, ?FUNCTION_NAME, ?FUNCTION_ARITY}
                , line => ?LINE
            }));
        false ->
            ok
    end).
```